euphony/src/main/java/co/euphony/rx/AudioRecorder.java

```diff
@@ -66,8 +73,8 @@ public void start() {
 	 */
 	public void stop() {
 		if (running) {
-			audioRecord.stop();
-			running = false;
+				audioRecord.stop();
+				running = false;
```

Comment:
```suggestion
			audioRecord.stop();
			running = false;
```

indent should be modified here.